app/controllers/stocks_controller.rb

```diff
@@ -2,7 +2,54 @@ class StocksController < ApplicationController
   before_action :authenticate_user!
   before_action :set_stock, only: [:update, :edit, :destroy]
   def index
-    @stocks = Stock.includes(:rawmaterial, { rawmaterial: [:foodcategory, :unit] }).where(user_id: current_user.id)
+    @stocks = current_user.stocks.includes(:rawmaterial, { rawmaterial: [:foodcategory, :unit] })
+    @todaysmenus = current_user.todaysmenus.includes(:cuisine, cuisine: :foodstuffs).search_in_today
+    # stockが全て消費される予定かどうかをbooleanでViewに送る
+    stocks_rawmaterial_ids_and_quantites = @stocks.pluck(:rawmaterial_id, :quantity)
```

Comment:
読みやすい変数名がイメージがつかない(1)

Reply:
別にカラム名まで変数に記載しなくていいと思います
```suggestion
    stocks = @stocks.pluck(:rawmaterial_id, :quantity)
```

Reply:
そもそも変数に格納しなくていいかもです